custom_components/echonetlite/climate.py

```diff
@@ -108,7 +109,7 @@ def device_info(self):
             "identifiers": {
                   (DOMAIN, self._connector._uid, self._connector._instance._eojgc, self._connector._instance._eojcc, self._connector._instance._eojci)
             },
-            "name": EOJX_CLASS[self._connector._instance._eojgc][self._connector._instance._eojcc]
+            "name": self._device_name
```

Comment:
This stops issues with multiple devices all being called HomeAirConditioner which was a) confusing and b) causing issues when adding more than one item.